core/src/main/java/org/pentaho/di/connections/utils/EncryptUtils.java

```diff
@@ -91,14 +109,9 @@ public static String getValue( Object object, Field field ) {
     try {
       Method getMethod =
         object.getClass().getMethod( GET_PREFIX + StringUtils.capitalize( field.getName() ) );
-      if ( getMethod != null ) {
-        return (String) getMethod.invoke( object );
-      } else {
-        return null;
-      }
+      return (String) getMethod.invoke( object );
```

Comment:
Are we missing an edge case, where previously it was returning null if `getMethod` was null ?

```
if ( getMethod != null ) {
        return (String) getMethod.invoke( object );
      } else {
        return null;
      }
```

Reply:
Removed because `getMethod` is never `null`.